<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[reading devurl from property file]]>
    </line>
    <line>
      <![CDATA[reading devUserId from property file]]>
    </line>
    <line>
      <![CDATA[wetting for1000ms]]>
    </line>
    <line>
      <![CDATA[sending user name]]>
    </line>
    <line>
      <![CDATA[wetting for1000ms]]>
    </line>
    <line>
      <![CDATA[sign in click....]]>
    </line>
    <line>
      <![CDATA[reading devPassword from property file]]>
    </line>
    <line>
      <![CDATA[wetting for2000ms]]>
    </line>
    <line>
      <![CDATA[enter passworrd successfully....]]>
    </line>
    <line>
      <![CDATA[wetting for1000ms]]>
    </line>
    <line>
      <![CDATA[click on password submit....]]>
    </line>
    <line>
      <![CDATA[wetting for1000ms]]>
    </line>
    <line>
      <![CDATA[wetting for1000ms]]>
    </line>
    <line>
      <![CDATA[handling popup......]]>
    </line>
    <line>
      <![CDATA[getting actual username]]>
    </line>
    <line>
      <![CDATA[reading devexpUserId from property file]]>
    </line>
    <line>
      <![CDATA[logout apllication.....]]>
    </line>
  </reporter-output>
  <suite started-at="2022-12-02T20:24:11 IST" name="Default suite" finished-at="2022-12-02T20:25:10 IST" duration-ms="59001">
    <groups>
    </groups>
    <test started-at="2022-12-02T20:24:11 IST" name="Default test" finished-at="2022-12-02T20:25:10 IST" duration-ms="59001">
      <class name="neoStoxTesting.NeoStoxUsernameValidating">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:neoStoxTesting.NeoStoxUsernameValidating@729d991e]" started-at="2022-12-02T20:24:11 IST" name="launchBrowser" finished-at="2022-12-02T20:24:25 IST" duration-ms="13573" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[reading devurl from property file]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="signIntoBrowser()[pri:0, instance:neoStoxTesting.NeoStoxUsernameValidating@729d991e]" started-at="2022-12-02T20:24:25 IST" name="signIntoBrowser" finished-at="2022-12-02T20:25:09 IST" duration-ms="44322" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[reading devUserId from property file]]>
            </line>
            <line>
              <![CDATA[wetting for1000ms]]>
            </line>
            <line>
              <![CDATA[sending user name]]>
            </line>
            <line>
              <![CDATA[wetting for1000ms]]>
            </line>
            <line>
              <![CDATA[sign in click....]]>
            </line>
            <line>
              <![CDATA[reading devPassword from property file]]>
            </line>
            <line>
              <![CDATA[wetting for2000ms]]>
            </line>
            <line>
              <![CDATA[enter passworrd successfully....]]>
            </line>
            <line>
              <![CDATA[wetting for1000ms]]>
            </line>
            <line>
              <![CDATA[click on password submit....]]>
            </line>
            <line>
              <![CDATA[wetting for1000ms]]>
            </line>
            <line>
              <![CDATA[wetting for1000ms]]>
            </line>
            <line>
              <![CDATA[handling popup......]]>
            </line>
          </reporter-output>
        </test-method> <!-- signIntoBrowser -->
        <test-method signature="validateUserName()[pri:0, instance:neoStoxTesting.NeoStoxUsernameValidating@729d991e]" started-at="2022-12-02T20:25:09 IST" name="validateUserName" finished-at="2022-12-02T20:25:09 IST" duration-ms="45" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[test is failed exp and actual user id not matching expected [Hi VCTC_09_Jul C_Mahesh Vikas Patil......] but found [Hi VCTC_09_Jul C_Mahesh Vikas Patil]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: test is failed exp and actual user id not matching expected [Hi VCTC_09_Jul C_Mahesh Vikas Patil......] but found [Hi VCTC_09_Jul C_Mahesh Vikas Patil]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at neoStoxTesting.NeoStoxUsernameValidating.validateUserName(NeoStoxUsernameValidating.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[getting actual username]]>
            </line>
            <line>
              <![CDATA[reading devexpUserId from property file]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateUserName -->
        <test-method is-config="true" signature="AfterMethod()[pri:0, instance:neoStoxTesting.NeoStoxUsernameValidating@729d991e]" started-at="2022-12-02T20:25:09 IST" name="AfterMethod" finished-at="2022-12-02T20:25:10 IST" duration-ms="483" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logout apllication.....]]>
            </line>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:neoStoxTesting.NeoStoxUsernameValidating@729d991e]" started-at="2022-12-02T20:25:10 IST" name="afterClass" finished-at="2022-12-02T20:25:10 IST" duration-ms="514" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- neoStoxTesting.NeoStoxUsernameValidating -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
